### ISSUE TITLE
(2.1.2 pre2) Can't dismount with Warclaw in WvW with smart warclaw

### ISSUE THREAD

**Issue by itVictor** [Friendly0Fire/GW2Radial#270]
**Describe the bug**
Pretty self explanatory, Enabling the smart warclaw option and then trying to dismount it with the mount button does nothing. It worked fine in 2.1.1.

**To Reproduce**
1. Enable smart warclaw
2. go to wvw and get on warclaw
3. notice that pressing mount button (to dismount) does nothing

**Expected behavior**
Pressing mount button while on warclaw in wvw should dismount normally.

**Screenshots**
n/a

**System configuration (please fill in):**
 - OS: Windows 10
 - GPU: 5700xt
 - Chainloading arcdps -> GW2Radial

**Additional context**
No errors in the log from what I can see.

**Comment by Friendly0Fire** [comment]
I'm unable to reproduce on my end. Are you sure you've got your keybinds setup correctly?

**Comment by itVictor** [comment]
Tried it again and it seems to work now, not sure what I did differently (maybe settings didn't get saved when I updated the first time?).

I'll reopen the issue if it happens again/can figure out how to reproduce.

**Comment by itVictor** [comment]
@Friendly0Fire I was able to reproduce it and I believe I found the problem.

If you go DIRECTLY into WvW after logging in (without doing anything else), you are unable to dismount in WvW. However, if you open the wheel at any point OUTSIDE of WvW and then go back into WvW, the dismount works fine. So we have:

1. Select character, open the wheel in PvE, go into WvW, dismount fine (good)
2. Select character, go directly into WvW, unable to dismount (broken)
3. Select character, go directly into WvW, unable to dismount, exit WvW, open the wheel in PvE, go back into WvW, dismount fine (partially broken)

So in short, if the first time the wheel is initialized is in WvW (presumably with quick warclaw and quick dismount active in settings), you won't be able to dismount until you open the menu anywhere else in the game.

Here's my config.ini if it helps: https://pastebin.com/Sxw2tipg

**Comment by Friendly0Fire** [comment]
I'm afraid I cannot reproduce even with your config. Make sure *all* of your mount keybinds are properly set, the addon will not always use the same keybind to dismount.

I made a minor tweak that could assist with that scenario, you can look [here](https://github.com/Friendly0Fire/GW2Radial/actions/runs/1727109423) for the corresponding nightly.

**Comment by itVictor** [comment]
The issue persisted even after deleting everything and doing a clean install.

However I've tried the nightly build and it seems to have resolved it.
